api/apps/api/src/modules/projects/blm/set-project-blm-handler.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { SetProjectBlmHandler } from '@marxan-api/modules/projects/blm/set-project-blm-handler';
+import { ProjectBlmRepo } from '@marxan-api/modules/blm';
+import { SetProjectBlm } from '@marxan-api/modules/projects/blm/set-project-blm';
+import { right } from 'fp-ts/Either';
+import { Test } from '@nestjs/testing';
+import { MemoryProjectBlmRepository } from '@marxan-api/modules/blm/values/repositories/memory-project-blm-repository';
+
+describe('set-project-blm-handler', () => {
+  const projectId = 'fooId';
+  let blmRepository: ProjectBlmRepo;
+  let setProjectBLMHandler: SetProjectBlmHandler;
+
+  beforeEach(async () => {
+    const module = await Test.createTestingModule({
+      providers: [
+        {
+          provide: ProjectBlmRepo,
+          useFactory: () => new MemoryProjectBlmRepository(),
```

Comment:
[minor] Could be just `useClass: MemoryProjectBlmRepository`

Reply:
Not really sure why, but using `useClass` throws an error about dependencies